test/arsi_test.rb

```diff
@@ -85,6 +85,18 @@
     end
   end
 
+  describe "when arsi is disabled" do
+    before do
+      Arsi::UnscopedSQL.expects(:sql_check!).never
+      Arsi.disable!
+    end
+
+    it "does not call sql_check if disabled" do
+      assert User.where(:password => 'hello').delete_all
+      Arsi.enable!
+    end
```

Comment:
better split of concerns:
```suggestion
    around do |t|
      Arsi.disable!
      t.call
    ensure
      Arsi.enable!
    end

    it "does not call sql_check if disabled" do
      Arsi::UnscopedSQL.expects(:sql_check!).never
      assert User.where(password: 'hello').delete_all
    end
```

Reply:
💚

Reply:
Oh, `around` is rspec

Reply:
use minitest-around or maxitest

Reply:
https://github.com/zendesk/arsi/pull/20